### ISSUE TITLE
EthernetESP32 example

### ISSUE THREAD

**Issue by AliReshadF** [Networking-for-Arduino/EthernetESP32#4]
Hello

Today I  attempted to run HelloServer.ino from the example folder and uploaded this code to my ESP32. Inside the serial monitor, I can see my device obtaining an IP through DHCP and it also prints MDNS responder started successfully. Here is what I got in the serial monitor:

```
Initialize Ethernet with DHCP:
DHCP assigned IP 192.168.1.102
MDNS responder started
HTTP server started
```
I can also access the 192.168.1.102 address and see my webpage but when I use software like [NETworkManager](https://github.com/BornToBeroot/NETworkManager/releases) or AngryIP scanner, I cannot find my device hostname. I would appreciate it if you could help me.

More information :
Development board = DOIT ESP 32 Devkit V1
Arduino core version = 3.0.5

**Comment by JAndrassy** [comment]
and with the WiFi version of that example you can find the hostname?

**Comment by AliReshadF** [comment]
> and with the WiFi version of that example you can find the hostname?

Yes, I can find it inside of both software. I also should mention I am using enc28j60.

**Comment by JAndrassy** [comment]
maybe mdns uses WiFi if it is available. is WiFi off?

**Comment by AliReshadF** [comment]
In my code, there is no code for wifi. I didn't even include wifi libraries. Everything is as same as [HelloServer.ino](https://github.com/Networking-for-Arduino/EthernetESP32/blob/master/examples/HelloServer/HelloServer.ino) example. Is there any special way to turn off wifi?

**Comment by JAndrassy** [comment]
esp32 can start the WiFi at boot and connect to the last AP with stored ssid and password.

**Comment by AliReshadF** [comment]
> esp32 can start the WiFi at boot and connect to the last AP with stored ssid and password.

I have checked my router to find out which devices are connected to the router. My esp32 connected via ethernet not wifi. now is there any suggestion?
